sdks/dart/client-dart-sdk/README.md

```diff
@@ -115,3 +115,58 @@ enum Readyness {
 }
 
 ```
+
+## Mobile API
+
+FeatureHub also has a mobile friendly API, that is intended to be used like a GET request and disconnect. The
+point here is that the Mobile devices radio does not stay on like it would with an EventSource. For Dart server 
+instances, client side apps (such as Flutter for Web, Linux, Windows or Mac OS X) - you should use the Event Source
+above. For Flutter for Android or iOS, please use this API.
+
+It is simple to use, you need to specify the host base url and the environment(s) that you wish to pull into your 
+application. Do not have features with the same names otherwise you will encounter issues with versioning.
+
+Construction is fairly simple, you need a repository and there is an example in `example/dart_cli/get_main.dart`.
+
+```dart
+final es = FeatureHubSimpleApi(sdkHost, [sdkUrl], repo);
```

Comment:
what is sdkHost? normally it a part of sdkUrl?

Reply:
no, its the Edge url, so i better explain better.

Reply:
oh ok, but where do I find this edge host? is it something quite obvious?